packages/config/src/validate/validations.js

```diff
@@ -212,6 +219,14 @@ const POST_NORMALIZE_VALIDATIONS = [
       functions: { directory: 'my-functions' },
     }),
   },
+  {
+    property: 'functions.*.schedule',
+    check: isCronExpression,
+    message: 'must be valid cron syntax.',
```

Comment:
I wonder if we can add a link here that points to additional information about the cron syntax.

Reply:
Some links that could be helpful:

https://github.com/harrisiirak/cron-parser (the library we use for parsing)
https://crontab.guru/ (super helpful website that explains cron syntax, but not necessarily 100% the same as our parser)

Reply:
Sounds good! I'd create a shortlink like `https://ntl.fyi/cron-syntax` and point it to whichever resource we choose. I can show you how to create one of these.

Reply:
Done!